pyproject.toml

```
@@ -81,6 +81,7 @@ fs-s3fs = {version = ">=1.1.1", optional = true}
 # Testing dependencies installed as optional 'testing' extras
 pytest = {version=">=7.2.1", optional = true}
 pytest-durations = {version = ">=1.2.0", optional = true}
+backports-datetime-fromisoformat = "^2.0.1"
```

We can avoid installers building this package from the source distribution on Python 3.11+ since [there are no wheels for those](https://pypi.org/project/backports-datetime-fromisoformat/#files):

```suggestion
backports-datetime-fromisoformat = { version = ">=2.0.1", python = "<3.11" }
```

And you'll probably want to run `poetry lock --no-update` after that 😉.